### PR TITLE
Review.rst: Add a note to reviewers

### PR DESCRIPTION
**Pull request by ShubhamBansal1997**
Issue number #5420

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by jayvdb** [comment]
Please squash your commits and follow http://coala.io/commit

**Comment by ShubhamBansal1997** [comment]
@jayvdb  Please check is it according to the commit guidelines.

File: docs/Developers/Review.rst [hunk 1]
```diff
@@ -132,6 +132,8 @@ For the Reviewers
    Bears require special attention during testing.
    `Testing Bears <http://api.coala.io/en/latest/Developers/Testing_Bears.html>`__
    provides a guideline for how to test bears.
+-  Check that the changes haven't been pushed from master branch.
+   Although it's not mandatory but a good practice to follow.
```
Comment (jayvdb) [inline docs/Developers/Review.rst]: This sentence needs to be improved so it is easier to understand.

Comment (yashovardhanagrawal) [reply docs/Developers/Review.rst]: You might change it to `As a good practice, check if the changes in PR are not pushed from the master branch`

Comment (ShubhamBansal1997) [reply docs/Developers/Review.rst]: @jayvdb  Sentence updated as per mentioned by @yashovardhanagrawal. Please merge the PR

**Comment by ShubhamBansal1997** [comment]
@jayvdb removed "update doc"

File: docs/Developers/Review.rst [hunk 2]
```diff
@@ -132,6 +132,8 @@ For the Reviewers
    Bears require special attention during testing.
    `Testing Bears <http://api.coala.io/en/latest/Developers/Testing_Bears.html>`__
    provides a guideline for how to test bears.
+-  As a good practice, check if the changes in PR are not pushed
+   from the master branch
```
Comment (jayvdb) [inline docs/Developers/Review.rst]: full stop at end of your sentence.

what does "As a good practice" mean for the reviewer?

what should be done if the PR is from the master branch?

**Comment by li-boxuan** [comment]
Closed due to inactivity.

**Pull request closed without merging**